cpp/src/aztec/proof_system/flavor/flavor.hpp

```diff
@@ -36,7 +36,8 @@ struct StandardHonk {
   public:
     using Arithmetization = proving_system::StandardArithmetization;
     using MULTIVARIATE = Arithmetization::POLYNOMIAL;
-    static constexpr size_t STANDARD_UNROLLED_MANIFEST_SIZE = 12; // cf waffle::STANDARD_UNROLLED_MANIFEST_SIZE
+    // TODO(Cody): Where to specify? is this polynomial manifest size?
+    static constexpr size_t STANDARD_HONK_MANIFEST_SIZE = 16;
```

Comment:
Can this just be deleted? Not currently used anywhere. Or should the value be specified here then used by the polynomial manifest?

Reply:
I left it in only as a comment, as a suggestion that perhaps this data should be specified as part of the flavor.